src/stest/tests/set-up-block-special-file

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+# main : () -> string
+# Return a path to a block special file that exists on most systems.
+function main {
+	echo "/dev/disk/by-diskseq/1" || return 1
```

Comment:
I did my best to come up with a file I thought would be an most unix machines, but I'm not the most familiar with linux and don't know how well this works for sure.

Maybe there's an easy way to construct a block special file that we can use instead?

Reply:
I __believe__ that `/dev/disk/by-diskseq/1` works here. Good job finding this.